flink-connectors/flink-connector-pulsar/archunit-violations/stored.rules

```diff
@@ -0,0 +1,4 @@
+#
+#Thu Mar 03 12:42:13 CST 2022
+Tests\ inheriting\ from\ AbstractTestBase\ should\ have\ name\ ending\ with\ ITCase=3ac3a1dc-681f-4213-9990-b7b3298a20bc
```

Comment:
Do you know why this references an empty file? @JingGe

Reply:
It is fine, because there is no violation for this rule and the store need to be intialized.